File: android-lib-retrofit/retrofit-sample/app/src/main/java/com/chiclaim/android/retrofit_sample/source/GitHub.java
package com.chiclaim.android.retrofit_sample.source;

import com.chiclaim.android.retrofit_sample.bean.Contributor;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

import java.util.List;

public interface GitHub {
    @GET("/repos/{owner}/{repo}/contributors")
    Call<List<Contributor>> contributors(
            @Path("owner") String owner,
            @Path("repo") String repo);
}

File: android-archietecture/jetpack/jetpack-sample/app/src/main/java/com/chiclaim/jetpack/repository/Github.kt
package com.chiclaim.jetpack.repository

import androidx.lifecycle.LiveData
import com.chiclaim.jetpack.bean.Contributor
import retrofit2.http.GET
import retrofit2.http.Path


interface Github {

    @GET("/repos/{owner}/{repo}/contributors")
    fun contributors(
            @Path("owner") owner: String?,
            @Path("repo") repo: String?): LiveData<List<Contributor?>?>?
}